### DISCUSSION TITLE
Violentmonkey and uMatrix: supported combo?

### DISCUSSION THREAD

**Discussion by farblos** [violentmonkey/violentmonkey#1478] [Q&A]
This has been nagging me for quite a while since it seems to be inherently contradictory:

Does VM support running user scripts on some site A even if JavaScript on site A is banned by means of uMatrix?

It actually seems to work, but is that by coincidence or something that I can rely on?

OT: Q: Why would one need that? A: For example, to always load lazy-loaded images on some sites.

**Comment by tophf** [comment]
If it works, it works. Apparently one extension can't apply this restriction to another extension.
